docs.feldera.com/docusaurus.config.ts

```diff
@@ -67,14 +67,206 @@ const config: Config = {
   ],
 
   plugins: [
+    [
```

Comment:
is this temporary?
mark it with a "to delete"?

Reply:
don't think it needs to be deleted at least not anytime soon